src/modules/nft.ts

```diff
@@ -193,9 +193,29 @@ export class NFTModule extends ModuleWithRoles<NFT> {
   }
 
   public async setRoyaltyBps(amount: number): Promise<TransactionReceipt> {
```

Comment:
You got the nft module, nice. But we also need to update the pack, drop, and bundle modules. I'll take care of this, but please note that you only implemented the fix for 1 out of our 4 modules.